lib/assembly/broadway.ex

```diff
@@ -29,6 +30,15 @@ defmodule Assembly.Broadway do
   @impl true
   @decorate transaction(:queue)
   def handle_message(_, %Message{} = message, _context) do
+    %{resource: resource, request_id: request_id} =
+      message
+      |> URI.decode()
+      |> Bottle.Core.V1.Bottle.decode()
+
+    Logger.metadata(request_id: request_id)
+
+    handle_resource(resource)
```

Comment:
I think this should be the last line so we handle errors correctly no?

Reply:
Yup. I'll update this to follow the pattern used in our other services.